homeassistant/components/cover/zwave.py

```diff
@@ -96,6 +96,8 @@ def update_properties(self):
     @property
     def is_closed(self):
         """Return if the cover is closed."""
+        if self.current_cover_position is None:
+            return
```

Comment:
If current_position is not populated, mean no data from zwave device is fetched. Therefore we return none, and that becomes a state of unknown: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/cover/__init__.py#L181